texi2rst.py

```diff
@@ -1216,7 +1216,8 @@ def __init__(self, table_element, debug):
             for idx in range(self.num_columns):
                 column = []
                 for row in comp.rows:
-                    column.append(row.entries[idx])
+                    if idx < len(row.entries):
+                        column.append(row.entries[idx])
```

Comment:
Why did you need this?  Can we have a unit test please?

Reply:
I'll do it in a different pull request, I accidentally pushed a work-in-progress version.

Reply:
I think this hunk isn't quite resolved yet

Reply:
You're right. I've added a test-case for it.